nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -163,7 +163,11 @@
 - module-name: 'arcade' # checksum: 3f9f0b7d
   data-files:
     dirs:
-      - 'resources'
+      - 'resources\system'
```

Comment:
That slash form ought to be wrong, although I can imagine it works on Windows, and this might be a Windows only package, so I might want to do something about this, such that auto-format cleans this up or something like that.

Reply:
Yeah, this is actually going to require so fixing on Arcade's side because it uses Windows style slashes everywhere, so doing it correctly here causes issues.

thank you for the heads up

Reply:
Ah, that's Nuitka internal, after changes I made it will just require you to change it over to `resources/system` and work fine. It's just because Windows accepts forward slashes, while Linux hates backlashes and os.path.normpath doesn't correct them there. You PR is not incorrect because of this, but I actually have checks for dest_path and the like that would have caught you well, if `posixpath.normpath` worked the way I think. I can switch this over with no effects. You code can do whatever it pleases in arcade.